pkg/cluster/builder.go

```diff
@@ -225,3 +226,27 @@ func buildEKSARelease(release *releasev1.EksARelease, bundle *releasev1.Bundles)
 	}
 	return eksaRelease
 }
+
+// GetBundlesManifest builds a Bundles from a FileSpecBuilder.
+func GetBundlesManifest(b FileSpecBuilder) (*releasev1.Bundles, error) {
```

Comment:
why do we use a `FileSpecBuilder` to read a bundles?
`FileSpecBuilder` is to build a cluster.Spec
Do we need any of the logic inside the builder? or can we use the `manifests` package directly